peribolos/knative-extensions.yaml

```diff
@@ -1188,6 +1200,18 @@ orgs:
         - nak3
         - skonto
 
+      serving-progressive-rollout Approvers:
+        description: Approver group for serving-progressive-rollout - to be used in CODEOWNERS file
+        privacy: closed
+        repos:
+          serving-progressive-rollout: write
+        members:
+          - houshengbo
+          - yuzisun
+          - sukumargaonkar
+          - dprotaso
+          - skonto
```

Comment:
you can drop my name and skonto - since when you make the owners file you can add `serving-writers` as a separate entry

eg. see
https://github.com/knative-extensions/net-istio/blob/1252a2453f21dd19795f8d016a4ea271e213eca2/OWNERS#L3-L11

Reply:
ok